scripts/west_commands/genboard/templates/nrf52/board.yml.jinja2

```diff
@@ -0,0 +1,5 @@
+board:
```

Comment:
isn't part of the purpose of `genboard/config.yml` to have the information needed to create the board.yml.

afaict, the `config.yml` already contains information regarding SoC name, CPU clusters (cores), non-secure variant, and thus there shouldn't be a need to maintain a board.yml per SoC series.

Reply:
we need mainly for:

1. "variant" information (e.g. laca, qkaa, etc.) which indicates packages and memories
2. ram/rom, for twister files
3. "since" info, ie, since when the soc is supported

I guess we could re-use upstream files by _extending_ them, but I left this for a 2nd iteration.

Reply:
> I guess we could re-use upstream files by extending them, but I left this for a 2nd iteration.

accepted.